x-pack/legacy/plugins/reporting/export_types/csv_from_savedobject/server/lib/generate_csv_search.ts

```diff
@@ -80,10 +80,11 @@ export async function generateCsvSearch(
 
   let payloadQuery: QueryFilter | undefined;
   let payloadSort: any[] = [];
+  let docValueFields: any[] | undefined;
   if (jobParams.post && jobParams.post.state) {
     ({
       post: {
-        state: { query: payloadQuery, sort: payloadSort = [] },
+        state: { query: payloadQuery, sort: payloadSort = [], docvalue_fields: docValueFields },
```

Comment:
@elastic/kibana-app can you help us understand why the `docvalue_fields` info is part of the `state` that we get from the panel info? It means the id of the saved search doc isn't enough to derive an actual ES query - the state data is additional information.